update_deps/build_graph.ps1

```diff
@@ -51,6 +51,11 @@ function get-name-from-url {
     param (
         [string] $url
    )
+   if(!$url.Contains("http"))
```

Comment:
I wonder why this was needed. It should not be needed, right?

Reply:
From the PR description:

"The script would accept a file path as -root. This is undesirable because the script would still work in most cases, unless the remotes for some repositories were set to not be Azure or Github. These changes validate that the value passed to the -root argument is a valid URL. "